### PR TITLE
fix(core): fixed envIdentifier issue in workflow-creator

### LINKED ISSUE #24: How to pass `envIdentifier` with Vanilla JS
**Describe the bug**
Once XML file is generated from `workflow-creator`, it uses `ENV_TOKEN` with Angular DI and adds it into XML file.
How do I provide `ENV_TOKEN` with Vanilla JS or React?
At the moment, in place of `ENV_TOKEN` value it's coming as undefined.

**Screenshots**
Screenshot 2023-08-21 at 6 34 20 PM
Screenshot 2023-08-21 at 6 34 42 PM

**Additional context**
Using the web component with React.js

### PR DESCRIPTION
**Pull request by arpit1503khanna**
envIdentifer now can be passed in vanillaJS/React by making envIdentifier a part of global window object

GH-24

## Description
envIdentifer now can be passed in vanillaJS or any other non angular environment by making envIdentifier a part of global window object. Implementation of the same can be seen in the README.md file 

Fixes #24 

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Intermediate change (work in progress)

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

## Checklist:

- [ ] Performed a self-review of my own code
- [ ] npm test passes on your machine
- [ ] New tests added or existing tests modified to cover all changes
- [ ] Code conforms with the style guide
- [ ] API Documentation in code was updated
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: projects/workflows-creator/src/lib/workflow-builder.module.ts [hunk 1]
```diff
@@ -1,18 +1,18 @@
...
-import {CAMUNDA} from './schema/camunda.json';
-import {BpmnElementService} from './services/bpmn/element.service';
-import {AutoLayoutService} from './layout/layout.service';
-import {DiFactoryService} from './layout/di.service';
-import {NgxPopperjsModule} from 'ngx-popperjs';
-import {StartElement} from './services/bpmn/elements/base/start.element';
-import {EndElement} from './services/bpmn/elements/base/end.element';
-import {GatewayElement} from './services/bpmn/elements/gateways/gateway.element';
-import {ProcessElement} from './services/bpmn/elements/process/process.element';
+import { BrowserModule } from '@angular/platform-browser';
```
Comment (yeshamavani) [inline projects/workflows-creator/src/lib/workflow-builder.module.ts:5]: please format this 
extra space

**Review by yeshamavani — CHANGES_REQUESTED** [review]

File: projects/workflows-creator/src/lib/workflow-builder.module.ts [hunk 2]
```diff
@@ -115,65 +115,77 @@ import {ENV_TOKEN} from './token';
...
+      ).workflowEnv,
+    },
     LocalizationPipe,
+    //{envIdentifier:window.envIdentifier}
+    //window.envIdentifier="local"
+    // useValue:"local"
+    // useValue: {envIdentifier: (window as Window & typeof globalThis & {envIdentifier:string}).envIdentifier}
+    //useValue:{objectType:Window}
+    // <script>window.workflowEnv = {envIdentifier:"heloo"}</script>
   ],
```
Comment (yeshamavani) [inline projects/workflows-creator/src/lib/workflow-builder.module.ts:189]: remove commented code.

**Review by yeshamavani — CHANGES_REQUESTED** [review]

File: projects/workflows-creator/src/lib/workflow-builder.module.ts (hunk 1 above)
Comment (arpit1503khanna) [reply projects/workflows-creator/src/lib/workflow-builder.module.ts]: done

File: projects/workflows-creator/src/lib/workflow-builder.module.ts (hunk 2 above)
Comment (arpit1503khanna) [reply projects/workflows-creator/src/lib/workflow-builder.module.ts]: done

**Review by yeshamavani — APPROVED** [review]

**Pull request merged**